app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -32,22 +32,25 @@
 				"bureau_de_change",	"money_transfer", "post_office", "library",
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
-				"doctors", "clinic", "pharmacy", "veterinary" };
+				"doctors", "clinic", "pharmacy", "veterinary",
+				"place_of_worship", "school", "townhall"};
```

Comment:
> because the app should not ask the surveyor about private places (= places the surveyor may not be allowed to enter)
  - well, they are are some cases where it is possible just looking at the entrance: a few steps (no) or a dedicated ramp (yes)
  - if you are a local, maybe you can enter or ask someone who knows; you are on the spot, on survey, you can try to determine this kind of information. Or just hide the quest, there is not so many schools per area.
  - on special days, like voting days, events, ... a lot of people can enter
> because the wheelchair-tag on a private place is not useful
  - it is useful  
  - for analysis and statistics
  - for people who go there (office, school...) for the first time (visitors, parents/relatives, voters...)